### DISCUSSION TITLE
Add `--workers` parameter to the dev command

### DISCUSSION THREAD

**Discussion by AliHezarpisheh** [fastapi/fastapi-cli#143] [Questions] [answered]
### First Check

- [x] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
fastapi dev --workers 4
```

### Description

**Feature Request: Add `--workers` Parameter to `fastapi dev` Command**

**Problem:**
Currently, the `fastapi dev` command does not accept the `--workers` parameter. In my development environment, I want to mimic a distributed environment similar to production, but I cannot achieve this with `fastapi dev`. Instead, I must use `fastapi run`, which is less ideal for development.

**Proposed Solution:**
Add support for the `--workers` parameter to the `fastapi dev` command. This would allow developers to run multiple workers in their development environment.

**Additional Context:**
I am open to implementing this feature and submitting a PR if approved by the maintainers.

### Operating System

Linux

### Operating System Details

Doesn't matter.

### Python Version

3

**Comment by YuriiMotov** [comment, marked as answer]
From the docs of Uvicorn (https://uvicorn.dev/deployment/#running-from-the-command-line):
```
   --workers INTEGER               Number of worker processes. Defaults to the
                                  $WEB_CONCURRENCY environment variable if
                                  available, or 1. Not valid with --reload.
```

Since `fastapi dev` runs server with `--reload` option, specifying number of workers is not supported by Uvicorn
